llvm/lib/Transforms/Scalar/IR2Vec-LOF/custom_loop_distribution/custom_loop_distribution.cpp

```diff
@@ -153,6 +153,7 @@ void custom_loop_distribution::initPipeCommunication(
     errs() << "Features populated END...\n";
     int *out;
     size_t size;
+    // MLRunner->evaluate<int  >();
```

Comment:
Please cleanup all of your commented codes.

Reply:
done